mypy/plugins/attrs.py

```diff
@@ -101,6 +101,11 @@ def argument(self, ctx: 'mypy.plugin.ClassDefContext') -> Argument:
                 from mypy.checkmember import type_object_type  # To avoid import cycle.
                 converter_type = type_object_type(converter.node, ctx.api.named_type)
             elif converter and isinstance(converter.node, OverloadedFuncDef):
+                if converter.node.type is None:
+                    # In certain cases the final overload type is not yet ready.
+                    # We analyze it and schedule another pass.
+                    ctx.api.analyze_overloaded_func_def(converter.node)
```

Comment:
Can we use something less-specific like `converter.node.accept(ctx.api)`?

I am asking, because I don't like that we are adding `analyze_overloaded_func_def` as a part of the intreface 🙁

Reply:
`SemanticAnalyzerPluginInterface` isn't a `StatementVisitor`, so this doesn't type check :(

Reply:
Can we analyze the whole `converter`? Like `ctx.api.accept(converter)`?